packages/files-ui/cypress/tests/sharing-explainer-spec.ts

```diff
@@ -0,0 +1,55 @@
+import { navigationMenu } from "../support/page-objects/navigationMenu"
+import { sharingExplainerModal } from "../support/page-objects/modals/sharingExplainerModal"
+
+describe("Sharing Explainer", () => {
+
+  context("desktop", () => {
+
+    it("User can view and dismiss the sharing explainer", () => {
+      // intercept and stub the response to ensure the explainer is displayed
+      cy.intercept("GET", "**/user/store", {
+        body: [{ "csf.dismissedSharingExplainer": "false" }]
+      })
+
+      cy.web3Login()
+      navigationMenu.sharedNavButton().click()
+      sharingExplainerModal.body().should("be.visible")
+
+      // set up a spy for the POST response
+      cy.intercept("POST", "**/user/store").as("storePost").then(() => {
+
+        // dismiss the sharing explainer
+        sharingExplainerModal.nextButton().click()
+        sharingExplainerModal.nextButton().click()
+        sharingExplainerModal.gotItButton().click()
+        sharingExplainerModal.body().should("not.exist")
+
+        // intercept POST to ensure the key was updated after the explainer is dismissed
+        cy.wait("@storePost").its("request.body").should("contain", {
+          "csf.dismissedSharingExplainer": "true"
+        })
+      })
+    })
+
+    it("User should not see sharing explainer if previously dismissed", () => {
+      cy.intercept("GET", "**/user/store", {
+        body: [{ "csf.dismissedSharingExplainer": "true" }]
```

Comment:
it's a small thing that made a diff.
```suggestion
        body: { "csf.dismissedSharingExplainer": "true" }
```